freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/config/FreeIpaBackupConfigView.java

```diff
@@ -75,6 +75,7 @@ public String getAzureInstanceMsi() {
         map.put("hourly_enabled", this.hourlyEnabled);
         map.put("initial_full_enabled", this.initialFullEnabled);
         map.put("platform", ObjectUtils.defaultIfNull(this.platform, EMPTY_CONFIG_DEFAULT));
+        map.put("azure_instance_msi", ObjectUtils.defaultIfNull(this.azureInstanceMsi, EMPTY_CONFIG_DEFAULT));
```

Comment:
Yes, this fixes it (tested with cross-subscription setup).

One note though: in the freeipa_backup.sh command `/usr/local/bin/azcopy login --identity --identity-resource-id [azure-msi]` will report auth ok even if azure-msi is "". It might make sense to add a log line what the used msi was. I know it can be read in the config file but could help clarity.